app/workers/queue_listener.rb

```diff
@@ -104,8 +109,14 @@ def create_payment(params)
     client.create_recurring_payment(params[:params])
   end
 
-  def create_payment(params)
-    client.create_recurring_payment(params[:params])
+  def subscribe_member(params)
+    page_name = ENV['AK_SUBSCRIPTION_PAGE_NAME']
+    byebug
```

Comment:
@osahyoun could you help me out figure out this one. I have an issue where `ENV['AK_SUBSCRIPTION_PAGE_NAME']` is always empty string. I have set it under test in `config/env.yml` as `'registration'`, and even if I explicitly set it in the spec, it's still empty string. You can run `rspec spec/requests/subscriptions_spec.rb:19` and you'll hit the `byebug` in `subscribe_member`, where if you `pp ENV` you might run into the same issue. Tell me if you do.